cpp/src/parquet/encryption/crypto_factory.h

```diff
@@ -41,6 +41,7 @@ struct PARQUET_EXPORT EncryptionConfiguration {
       : footer_key(footer_key) {}
 
   /// ID of the master key for footer encryption/signing
+  /// If uniform_encryption is True, will use this key to encrypt all columns as well.
```

Comment:
Nit: let's add a blank line between summary and description.
```suggestion
  /// \brief ID of the master key for footer encryption/signing
  ///
  /// If uniform_encryption is True, this key will encrypt all columns as well.
```